### ISSUE TITLE
 Remember Last Station

### ISSUE THREAD

**Issue by udavst** [Edzelf/Esp-radio#84]
Can I make the last station play when I switch on, which was played before shutting down?

**Comment by Edzelf** [comment]
Yes, save the preset number in the internal EEPROM (not too often, and only if different than the previous setting).
At startup you retrieve the preset and store it in "ini_block.newpreset".
I will include this feature in the coming ESP32 version of the radio.

**Comment by udavst** [comment]
And in this version you can make a patch? For the test) I really want this feature, especially when there is no screen.

**Comment by Edzelf** [comment]
I will try.  But it may take quite some time (other priorities..).

**Comment by udavst** [comment]
Thanks )

**Comment by Edzelf** [comment]
The ESP32 version will remember the settings.
